src/components/styles/YouTube.scss

```
@@ -0,0 +1,94 @@
+@import 'variables';
+
+#intro-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: auto;
+}
+
+#intro-title {
+  font: $font-title;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+#intro-subtitle {
+  font: $font-title;
+  font-size: 26px;
+  margin: 0;
+  color: #fff;
+}
+
+#youtube-computer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
```

it runs off the screen for me :( does this work for you? since this has a background image, it might be best to make the object have width/height ratio same as the svg
```suggestion
  height: calc(45vw * 15/23);
  width: 45vw;
```